docker-compose-production.yml

```diff
@@ -42,8 +40,6 @@ services:
     volumes:
       - ./configs/config-dev.json:/usr/share/app/configs/config.json
       - ./configs/keystore.jks:/usr/share/app/configs/keystore.jks
-      - ./docs/:/usr/share/app/docs
-      - /tmp/iudx/:/tmp/iudx/
     ports:
```

Comment:
Does the port need to be changed to 8443 here too? (I'm not very familiar w/ compose files)

Reply:
the first port denotes the outside port on which container app is exposed and the second port refers to actual port inside the container which the application is binded to. Yeah we can keep 8443 on outside too, that's what is in Readme and is the behavior when run through maven exec